src/applications/hca/helpers.jsx

```diff
@@ -98,6 +134,47 @@ export const facilityHelp = (
   </div>
 );
 
+export function fileHelp({ formContext }) {
+  if (formContext.reviewMode) {
+    return <p />;
```

Comment:
Is it possible to just return `null`, or does that do something weird?

Reply:
Cool, `null` seems to work. I had just copied this from another implementation of the file uploader.